projects/igniteui-angular/src/lib/core/enums.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * This enumeration is used to configure whether the date/time picker has an editable input with drop down
+ * or is readonly - the date/time is selected only through a dialog.
+ */
+export const enum InteractionMode {
+    DROPDOWN = 'dropdown',
```

Comment:
Make the enum values capitalized, not all CAPS. I will update the contributing.md with this guidance. 

```
 export const enum InteractionMode {
  DropDown = 'dropdown',
  Dialog = "dialog"
}
```